### ISSUE TITLE
Generate double loc item & miss some of the links in sitemap

### ISSUE THREAD

**Issue by tmclouisluk** [boazpoolman/strapi-plugin-sitemap#73]
### Describe the bug

I found that there is some links missed and double generate links in sitemap. Links are generated by URL Bundles

![a](https://user-images.githubusercontent.com/18186155/158430834-c1f74ecc-67e0-432f-ba89-bd75b82bc67a.PNG)

### Settings

URL Bundles
![2](https://user-images.githubusercontent.com/18186155/158430878-eb6aeb10-006f-46dc-a1cc-6eae363f4f3b.PNG)

In `config/plugins.js`
```
module.exports = ({ env }) => ({
  'sitemap': {
    enabled: true,
    config: {
      autoGenerate: true,
      allowedFields: ['id', 'uid', 'slug'],
      excludedTypes: [],
    },
  },
  publisher: {
    enabled: true,
  },
});
```

### Expected behavior

No duplicated loc items and all item in URL Bundles should be generated

### System

- Node.js version: 14
- Strapi version: 4.0.7
- Plugin version: 2.0.6
- Database: Postgresql 13
- Operating system:

**Comment by boazpoolman** [comment]
Hey @tmclouisluk 

Thanks for the issue submission.
I haven't seen this issue myself before.

Can you confirm that your PR (#74) fixes the issue?
I can't seem to understand why. Please elaborate.

**Comment by tmclouisluk** [comment]
> Hey @tmclouisluk
> 
> Thanks for the issue submission. I haven't seen this issue myself before.
> 
> Can you confirm that your PR (#74) fixes the issue? I can't seem to understand why. Please elaborate.

I found that strapi query will pick id randomly if you don't order it yourself. Therefore, in my case, since I have more than 100 items in URL Bundles, it will divide the links into parts. If picking id randomly, there will be some duplicated. That's the reason my bug happened. 

Hence, I just put orderby id simply in core.js. Then it works right now :)

**Comment by boazpoolman** [comment]
Thanks for the fix @tmclouisluk !

Released with [v2.0.7](https://github.com/boazpoolman/strapi-plugin-sitemap/releases/tag/2.0.7)
